src/prefect/settings/models/root.py

```diff
@@ -189,9 +189,8 @@ def post_hoc_settings(self) -> Self:
                 self.server.ui.api_url = self.api.url
```

Comment:
I haven't fully thought this through, but I think perhaps this should be "/api" as well here? So in practice, we'd get rid of the if statement one line up, and just unconditionally use "/api" whenever self.server.ui.api_url is None .

Thanks so much for addressing my suggestion so quickly by the way!